source_control/git.py

```diff
@@ -212,6 +212,19 @@
 - git: repo=https://github.com/ansible/ansible-examples.git dest=/src/ansible-examples refspec=+refs/pull/*:refs/heads/*
 '''
 
+RETURN = '''
+after:
+    description: last commit revision of the repository retrived during the update
```

Comment:
Instead of `retrieved` I'd put `checked out`. Fetch without checkout is a rather unusual case.